Source/ActionController.swift

```diff
@@ -175,12 +176,15 @@ open class ActionController<ActionViewType: UICollectionViewCell, ActionDataType
     open func addAction(_ action: Action<ActionDataType>) {
         if let section = _sections.last {
             section.actions.append(action)
+            collectionView.reloadData()
```

Comment:
please do this when the action controller is already presented. And looks like `self.calculateContentInset()` is missing here. Maybe you can move next block at the end of this function:

```swift
// To be moved to the bottom
if self.presentingViewController != nil {
     collectionView.reloadData()
     self.calculateContentInset()
}
```